### DISCUSSION TITLE
This user account is blocked.

### DISCUSSION THREAD

**Discussion by Arpitkandwal** [vercel/community#4106] [Account Issue] [answered]
### Question

I tried login to vercel for the first time, but it is giving me This user account is blocked. What is this problem? can anyone help me with this?

**Comment by oluiscabral** [comment]
I'm having the same problem... It's really weird

**Comment by amyegan** [comment, marked as answer]
Sorry you're having trouble with login. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps.

For more info, please check this guide: [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)

To open a ticket please use the appropriate direct channel found in this guide: [What is the best way to get support from Vercel?](https://vercel.com/guides/how-to-get-vercel-support#direct-channels)
